modules/katello_activation_key.py

```diff
@@ -194,22 +194,20 @@ def main():
         # copied keys inherit the subscriptions of the origin, so one would not have to specify them again
         # deleted keys don't need subscriptions anymore either
         if state == 'present':
-            if 'subscriptions' not in entity_dict:
-                subscriptions = []
```

Comment:
Surely this means I can't remove all subscription now?

Reply:
pass `subscriptions: []` in your playbook